openassessment/fileupload/tests/test_api.py

```diff
@@ -268,7 +268,7 @@ def test_download_url_with_unauthorized_key(self):
         self.assertEqual(404, download_response.status_code)
 
     def test_upload_download_with_accented_key(self):
-        self.set_key("noël.jpg")
+        self.set_key(u"noël.jpg")
         upload_url = self.backend.get_upload_url(self.key, self.content_type)
```

Comment:
This change makes sense to get the test passing, but I wonder if this is really something we want to support.  Do `boto` and S3 even allow non-ASCII unicode keys?